src/powershell/enable-rdp.ps1

```diff
@@ -28,3 +28,48 @@ if ($(Get-NetFirewallRule -Name $name).Enabled -ne $true) {
     Write-Error "[X] Failed to verify firewall rule: $name" -ErrorAction Stop
 }
 Write-Output "[*] Firewall rule successfully verified: $name"
+
+# Allow Terminal Server shadowing through the Windows firewall
+$name = "Remote Desktop - Shadow (TCP-In)"
+Write-Output "[ ] Configuring firewall rule: $name"
+Set-NetFirewallRule -DisplayName $name -Enabled True -Profile Private
+if ($(Get-NetFirewallRule -DisplayName $name).Enabled -ne $true) {
+    Write-Error "[X] Failed to verify firewall rule: $name" -ErrorAction Stop
+}
+Write-Output "[*] Firewall rule successfully verified: $name"
+
+# Enable Terminal Server shadowing
+# Set value to option 2:
+# "Configure Group Policy setting to not require the user’s consent"
+$name = "Shadow"
+$path = 'HKLM:\SYSTEM\CurrentControlSet\Control\Terminal Server\WinStations\RDP-Tcp'
+Write-Output "[ ] Configuring setting: $name"
+Set-ItemProperty -Path $path -Name $name -Value "2"
+if ($(Get-ItemProperty -Path $path -Name $name).Shadow -ne 2) {
+    Write-Error "[X] Failed to verify setting: $name" -ErrorAction Stop
+}
+Write-Output "[*] Setting successfully verified: $name"
+
+# Give the Administrator account full control of Remote Desktop Services
+# Set PermissionPreSet value to option 2:
+# "WINSTATION_ALL_ACCESS"
+$name = "Administrator RDP All Access"
+Write-Output "[ ] Configuring setting: $name"
+$result = Get-CimInstance -Namespace root\CIMV2\TerminalServices -ClassName Win32_TSPermissionsSetting -Filter 'TerminalName ="RDP-Tcp"' |
+        Invoke-CimMethod -MethodName AddAccount -Arguments @{AccountName="BUILTIN\Administrators"; PermissionPreSet="2"}
+if ($result.ReturnValue -ne 0) {
+    Write-Error "[X] Failed to verify setting: $name" -ErrorAction Stop
+}
+Write-Output "[*] Setting successfully verified: $name"
+
+# Enable Shadow Remote Desktop
```

Comment:
I believe this is the correct terminology.

```suggestion
# Enable Remote Desktop Shadowing
```

Reply:
@mcdonnnj resolved with [881105e](https://github.com/cisagov/windows-server-packer/pull/22/commits/881105ef37628dc8692092ee78cd3eda1588861a)